### DISCUSSION TITLE
Istio Envoy  - Java autoinstrumentation v1.1.0

### DISCUSSION THREAD

**Discussion by wtcampos** [open-telemetry/opentelemetry-java-instrumentation#2909] [Q&A]
Hello, 

Is there anyone with experience using Envoy on Istio and the Java Autoinstrumentation Agent? I'm facing a context propagation failure if I'm using `HTTP2`, if I change to `HTTP1.1`, it works fine.

**Comment by jkwatson** [comment]
Hi @wtcampos . This sounds like a question for the instrumentation repository, rather than the core repository. I'll move it over there for you.

**Reply by wtcampos** [reply]
Thanks!

**Reply by trask** [reply]
hey @wtcampos, does context propagation not work at all, or does it sporadically work/not work? what framework is serving the requests on the Java side (e.g. servlet, spring webflux/netty, etc)?

**Reply by trask** [reply]
let's discuss further in #2890

**Reply by wtcampos** [reply]
OK
